axelrod/tests/strategies/test_cycler.py

```diff
@@ -45,13 +46,31 @@ class TestCycler(TestPlayer):
 
         def test_strategy(self):
             """Starts by cooperating"""
-            for i in range(20):
-                responses = itertools.islice(itertools.cycle(cycle), i)
-            self.responses_test(responses)
+            match_len = 20
+            actions_cycle = _get_actions_cycle_against_cooperator(cycle)
+            test_actions = list(itertools.islice(itertools.cycle(actions_cycle), match_len))
+            self.versus_test(axelrod.Cooperator(), test_actions)
 
     return TestCycler
```

Comment:
No, sorry ignore me, I meant `return TestCycler` but that was a stupid question.